### DISCUSSION TITLE
Creating a stream with a VOD file

### DISCUSSION THREAD

**Discussion by studentAlanMorgan** [ant-media/Ant-Media-Server#6446] [Q&A]
How to create a stream from a VOD file?
in the menu there are only links to a direct video file and a frame, but none of these options can create a broadcast to receive an rtmp link.
ant 2.9

**Comment by studentAlanMorgan** [comment]
The issue is resolved.
interferes with ":" in the link after the domain

**Comment by studentAlanMorgan** [comment]
This solves the problem partially unfortunately. When creating a broadcast, it freezes a lot and shows 1 frame every 5-10 seconds.
If you set the bitrate, the speed increases slightly, but it turns out that this is not available for the community version and they cannot create a broadcast without a bitrate.
It's all very sad.

**Reply by Mohit-3196** [reply]
Hi @studentAlanMorgan,
It is not entirely clear to me.
Can you please let me know what you are trying to achieve and share a reproduce scenario?

### Short description
__Brief description of what happened__


### Environment
* Operating system and version:
* Java version: 
* Ant Media Server version:
* Browser name and version:

### Steps to reproduce
1. 
2. 
3.

### Expected behavior
__Put as much detail here as possible__

### Actual behavior
__Put as much detail here as possible__


Cheers,
Mohit

**Reply by studentAlanMorgan** [reply]
if you create a broadcast from a VOD file, it is terribly slow

**Reply by studentAlanMorgan** [reply]
![2024-07-05_224630](https://github.com/ant-media/Ant-Media-Server/assets/81875252/4a543af0-fe34-4a89-adf8-c5f242c3fe0d)
missing menu items when broadcasting a VOD file

**Comment by burak-58** [comment]
Thank you @studentAlanMorgan for your question and comments. Let me convert this to discussion so that you may discuss the question with AMS community.

**Comment by studentAlanMorgan** [comment]
as I understand:
The colon ":" is placed after the domain because, as planned, there should be a port there, but we remove the port by redirecting to port 80, in the end the ant removes the port, but forgets to remove the ":"

**Reply by Mohit-3196** [reply]
Hi @studentAlanMorgan,
When I create a playlist with a VOD file it works fine for me.
I don't see any such issues or slowness, etc.

Thank you.

**Reply by studentAlanMorgan** [reply]
https://github.com/ant-media/Ant-Media-Server/assets/81875252/87e7fe45-ed9c-4eef-8fe2-0f408cf7af3c
For me it’s the other way around. Also note that there are not enough control elements in the stream menu, a screenshot is attached above.

**Reply by Mohit-3196** [reply]
Hi @studentAlanMorgan,
I can see all the options for the control elements.
![Screenshot 2024-07-09 161122](https://github.com/ant-media/Ant-Media-Server/assets/86982446/98e6787d-7ffb-46ea-b4e8-65d9320e6f7e)

Also since it is a VOD streaming, you should play with HLS and not WebRTC because since it is a VOD broadcast, there is no need for ultra-low latency and therefore you should use HLS for a better experience.

Also, I recommend you optimize the settings for a better playback experience. You can follow [these instructions](https://antmedia.io/docs/guides/publish-live-stream/playlist/creating-playlist/#optimizing-playlist-for-better-playback-experience) for the same.

Thank you.

**Reply by studentAlanMorgan** [reply]
hmm... why don't I have these menu items then?
also missing"Extradata" information

**Reply by Mohit-3196** [reply]
I don't have access to your server :)
Maybe you have an issue with installation or server initialization.
